### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -17037,7 +17051,9 @@ idefinite->indefinite
 idel->idle
 idelogy->ideology
 idemopotent->idempotent
+idendified->identified
 idendifier->identifier
+idenfied->identified
 idenfifier->identifier
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:17057]:
```text
idendifiers->identifiers
idenfifiers->identifiers
```
* https://github.com/search?q=idendifiers&type=code
* https://github.com/search?q=idenfifiers&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -29442,6 +29469,7 @@ sequenzes->sequences
 sequetial->sequential
 sequeze->squeeze, sequence,
 sequnce->sequence
+sequnced->sequenced
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:29472]:
```text
sequncer->sequencer
sequncers->sequencers
```
* https://github.com/search?q=sequncer&type=code
* https://github.com/search?q=sequncers&type=code

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -31114,6 +31143,9 @@ streched->stretched
 streches->stretches
 streching->stretching
 strectch->stretch
+strecth->stretch
+strecthed->stretched
+strecthes->stretches
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:31148]:
```text
strecthing->stretching
```
* https://github.com/search?q=strecthing&type=code

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -33684,6 +33716,7 @@ typographc->typographic
 typpe->type
 typped->typed
 typpes->types
+typpically->typically
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:33719]:
```text
typpical->typical
```
* https://github.com/search?q=typpical&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
@vikivivi done, see [42c7b12](https://github.com/codespell-project/codespell/pull/2434/commits/42c7b12e8264c358572cda689eae5215856160e1)

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**